tasks/installer.py

```
@@ -91,3 +92,70 @@ def build_linux_script(ctx, flavor, version, bin_amd64, bin_arm64, output):
+                        line += "JobTimeoutSec=3000\n"
+                    if "Wants=" in line:
+                        line = line.replace(".service", "-exp.service")
+                    if "Type=" in line:
+                        line = "Type=oneshot\n"
+                    if "Conflicts=" in line:
+                        line = "Conflicts=datadog-agent.service\n"
+                    if "Before=" in line:
+                        line = "Before=datadog-agent.service\n"
+                    if (
+                        "Restart=" in line
+                        or "# " in line
+                        or "StartLimitInterval=" in line
+                        or "StartLimitBurst=" in line
+                    ):
+                        continue  # Skip line
+                    if "ExecStart=" in line:
+                        line += "ExecStart=/bin/false\n"
+                        line += "ExecStop=/bin/false\n"
+                else:
+                    if "Before=" in line:
+                        continue  # Skip line
+                    if "After=" in line:
+                        after_no_agent = line.split("After=")[1].replace("datadog-agent.service", "").strip()
+                        if len(after_no_agent) == 0:
+                            continue  # Skip line
+                        line = "After=" + after_no_agent + "\n"
+                    if "BindsTo=" in line:
+                        line = line.replace(".service", "-exp.service")
+
+                if "Alias=" in line:
+                    line = line.replace(".service", "-exp.service")
+                if "Description=" in line:
+                    line = line.replace("\n", "") + " Experiment\n"
+                line = line.replace("stable", "experiment")
+                experiment_file += line
+
+        if not check:
+            with open(experiment_path, 'w') as f:
+                f.write(experiment_file)
+        else:
+            with open(experiment_path) as f:
```

Are we sure this file exists? Otherwise you might crash here

---

Good call. We want to raise an exception in any case but it's better if we control it.